polkadot-protocol/introduction/polkadot-networks.md

```diff
@@ -0,0 +1,78 @@
+title: Development Networks
+description: This section contains the list of networks and resources available for testing purposes for developers in the Polkadot ecosystem. 
+---
+
+# Polkadot Development Networks
+
+## Introduction
+
+Development and testing are vital to building robust dApps, parachains, and network upgrades in the Polkadot ecosystem. This section provides a comprehensive overview of the networks and tools for developers to experiment, test, and refine their innovations in a risk-free environment before deploying them to production networks. 
+
+Whether you're a Polkadot core developer, parachain engineer, or application developer, the resources outlined here will help you ensure your solutions are thoroughly tested and ready for real-world usage. By leveraging these networks, developers can avoid the costs and risks associated with live deployments while testing functionalities such as governance, cross-chain messaging, and runtime upgrades.
+
+This guide covers the tools and networks designed to support a wide variety of development and testing needs in the Polkadot ecosystem, from local development nodes to community-run TestNets.
+
+## Network Overview
+
+Polkadot's development process is a well-structured path designed to ensure that upgrades and new features undergo thorough testing before reaching the live production networks. This diagram illustrates the typical progression of a Polkadot development cycle, starting from local environments and eventually reaching the Polkadot MainNet. 
+
+### Development Flow
+
+``` mermaid
+
+flowchart LR
+    id1[Local] --> id2[Westend] --> id4[Kusama] --> id5[Polkadot]  
+    id1[Local] --> id3[Paseo] --> id5[Polkadot] 
+```
+
+In the Polkadot ecosystem, various features can be explored without risking or spending tokens on production networks by utilizing testing tools like [Chopsticks](#chopsticks) and engaging with TestNets. 
+
+A typical journey through the Polkadot development process might look like this:
+
+1. **Local development node** - for core Polkadot developers, the process begins in a local environment where you can work on upgrades and experiment with new features using a local development node. This environment allows for rapid iteration and testing in an isolated setup without the need for interacting with external networks
+
+2. **Westend** - once the upgrades are ready, they are deployed to [Westend](#westend), Polkadot's primary TestNet. Westend is designed to mimic the conditions of a live network without the risk of using real tokens. You can extensively test features here to ensure that everything works as expected before rolling them out to Kusama or Polkadot
+
+3. **Paseo** - parachain developers often use Paseo, a community-run TestNet that mirrors Polkadot's runtime. Paseo provides parachain and dApp developers a space to test their applications with runtime environments that are closely aligned with Polkadot's without risking their projects on live networks
```

Comment:
I misled you, I'm sorry! 😬 

Paseo should be separate. We should say that there are two different development journeys, route one being Local > Westend > Kusama > Polkadot for testing core protocol features. Route two should be Local > Paseo > Polkadot for dapp and parachain developers to have a testing environment

Reply:
@0xLucca feel free to confirm or suggest alternative changes to what I said 😬

Reply:
Quoting from Paseo repository:
`Paseo is for Parachains and dapp builders to build and test their applications ontop of a stable testnet. The other testnet we have is Westend, which is for Parachains to perform protocol testing. Westend will be the environment to test new changes to the Polkadot relaychain and system chains, e.g. Coretime, Beefy, and Asycbacking.`